isg-503.coffee

```diff
@@ -21,9 +34,9 @@ module.exports =
 	yocto:
 		machine: 'isg-503'
 		image: 'balena-image-flasher'
-		fstype: 'img'
+		fstype: 'balenaos-img'
 		version: 'yocto-dunfell'
-		deployArtifact: 'balena-image-flasher-isg-503.img'
+		deployArtifact: 'balena-image-flasher-isg-503.balenaos-img'
 		compressed: true
 
```

Comment:
@RockSupport can you please specify if this board has:
- SD-CARD slot to boot flasher image for provisioning
- eMMC to boot image after initial provisioning
- Ethernet
- Wifi
- Bluetooth

Reply:
without Wifi and Bluetooth